### PR TITLE
Accept url in config

### LINKED ISSUE #61: Is it possible to pass URL in config?

### PR DESCRIPTION
**Pull request by deadcoder0904**
Closes #61

### CODE REVIEW

File: test.js [hunk 1]
```diff
@@ -13,6 +13,10 @@ test('v4 with impossible timeout', async t => {
 	t.false(await isOnline({timeout: 1}));
 });
 
+test('check url', async t => {
+	t.true(await isOnline({url: 'https://google.com'}));
```
Comment (sindresorhus) [inline test.js]: You're not actually testing the  `url` option here as the IP checks are faster. You need to mock those out.

File: index.js [hunk 2]
```diff
@@ -5,7 +5,7 @@ const pAny = require('p-any');
 const pTimeout = require('p-timeout');
 
 const appleCheck = async options => {
-	const {body} = await got('http://captive.apple.com/hotspot-detect.html', {
+	const {body} = await got(options.url, {
```
Comment (sindresorhus) [inline index.js]: You cannot just change the URL here. Look at the surrounding code. (https://github.com/sindresorhus/is-online/pull/67/files#diff-168726dbe96b3ce427e7fedce31bb0bcR15)

**Comment by sindresorhus** [comment]
This is not what was agreed on in #61:

> I think we should bring it back, but only try the those hostnames if all the 3 current checks fail.

**Comment by deadcoder0904** [comment]
Sorry then, probably have no idea how to do it. Both testing & url config :(

**Pull request closed without merging**